.github/workflows/release_notes.yml

```diff
@@ -0,0 +1,69 @@
+name: Generate Release Notes
+
+on:
+  pull_request:
+
+# on:
+#   push:
+#     branches:
+#       - main
+
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Generate release notes and environment variables
```

Comment:
Creating environment variables is how we add dynamic naming for the source file and branch names.